task1/Program.cs

```diff
@@ -45,5 +71,792 @@ public static decimal PerformCalculation(int r)
             decimal square = Pi * r * r;
             return square;
         }
+
+        public static int GetHouresFromSeconds(int seconds)
+        {
+            int hours = seconds / 3600;
+            return hours;
+        }
+
+        public static bool IsNumberBetweenGiven()
+        {
+            //
+            // Input three numbers.
+            //
+            int a = Convert.ToInt32(Console.ReadLine());
+            int b = Convert.ToInt32(Console.ReadLine());
+            int c = Convert.ToInt32(Console.ReadLine());
+
+            if ((b < a && b > c) || (b > a && b < c))
```

Comment:
If you have a bool expression, no need to compare the result with true or false.
You can return the result of the expression:
return (b < a && b > c) || (b > a && b < c);